edgelet/iotedge/src/config/import/old_config/mod.rs

```diff
@@ -192,7 +192,7 @@ agent:
   type: 'docker'
   env: {}
   config:
-    image: 'mcr.microsoft.com/azureiotedge-agent:1.0'
+    image: 'mcr.microsoft.com/azureiotedge-agent:1.4'
```

Comment:
@arsing can you confirm that we should make this change? (Or @and-rewsmith are you confident that we should make this change, as opposed to leaving this particular logic at "1.0"?)

Reply:
Based on logic I posted above I can't see how leaving it to default as 1.0 would be good:
> Did an investigation there and think it should be updated. It is the default for the aziotctl config import command (in case old yaml doesn't have agent) which will be used and fed into the new settings creation. Defaulting to 1.4 is appropriate.

So I think we should update.

Reply:
Ok. I looked at the code and I think I agree. This default config is added first, then the customer's existing config is added second--any key-value pairs found in the customer's config will override values with the same key in the default. So this will ensure the latest agent is set if the user's old config.yaml didn't specify it.